content/_data/changelogs/weekly.yml

```diff
@@ -12207,16 +12207,24 @@
       pull: 5425
       authors:
       - daniel-beck
+      references:
+      - pull: 5425
+      - url: https://www.jenkins.io/doc/book/managing/built-in-node-migration/
+        title: Built-In Node Name and Label Migration
+      - url: https://www.jenkins.io/doc/book/security/controller-isolation/
+        title: Do not build on the built-in node
       message: |-
         Replace the term "master" with "controller" (for the main Jenkins application) or "built-in node", as appropriate.
```

Comment:
Not keen that it says:  ' "built-in node" ', rather than ' "built-in" node '

Reply:
I think the confusion comes from this entry describing changes in the user interface descriptions, where the phrase "master" is often replaced by "built-in node".  In those cases, a string replacement of "master" by "built-in" would have made the message text less readable for the user.

The next entry in the changelog is describing the change in agent labeling.  Agent labeling changes "master" to "built-in".

Reply:
I guess it just does not clearly convey the message the first bullet relates to changes to the UI and documentation nomenclature referencing the "built in node" (formerly know as master) , while the second formally changes the name  - is it to "built-in" or "(built-in)"

Reply:
As far as I can tell from my experiments with 2.307, the new label is `built-in`.  I can write a label expression as `(built-in)`, but that is evaluating the expression as `("built-in")`, not as `"(built-in)"`.  When I attempt a label `"(built-in)"`, it does not match any node, while `built-in` matches the built-in node.

Reply:
I've attempted an alternate phrasing in 9653fdb443ea551e6d904fcae73da1128bf15d45

Reply:
I can't say I tried digging too deeply into this before posting the S/O reply or these remarks as I don't build locally on controller (ie: "master") ...

The release notes third bullet shows : "_URL to the built-in node (now /computer/(built-in)/_", which is also what I saw in the url when I checked. Now that I look more closely, my LTS-2.263.4 instance does indicate "${URL}/computer/(master)/"; my nodes (which is what I normally check) are "${URL}/computer/LNX_A/", no parentheses, so was not aware of the distinction.  

As long as it's clear to users it's called`built-in`, not "`(built-in)"` but displays as "${URL}/computer/(built-in)/".

Reply:
> 
> I've attempted an alternate phrasing in [9653fdb](https://github.com/jenkins-infra/jenkins.io/commit/9653fdb443ea551e6d904fcae73da1128bf15d45)

I believe that reads better. Will leave the debate whether including a warning is in or out to @daniel-beck and @MarkEWaite :
_... it should use the label "`built-in`"_. Using the built-in node is highly discouraged - see  [Controller Isolation](https://www.jenkins.io/doc/book/security/controller-isolation/)